opm/polymer/fullyimplicit/FullyImplicitCompressiblePolymerSolver.hpp

```diff
@@ -80,11 +80,23 @@ namespace Opm {
                 : rq(num_phases)
                 , rsSat(ADB::null())
                 , rvSat(ADB::null())
+                , fip()
             {
             }
+
+            enum FipId {
+                FIP_AQUA = BlackoilPropsAdInterface::Water,
+                FIP_LIQUID = BlackoilPropsAdInterface::Oil,
+                FIP_VAPOUR = BlackoilPropsAdInterface::Gas,
+                FIP_DISSOLVED_GAS = 3,
+                FIP_VAPORIZED_OIL = 4,
+                FIP_PV = 5,                    //< Pore volume
+                FIP_WEIGHTED_PRESSURE = 6
+            };
```

Comment:
Why is this enum duplicated? Also, why are pure ints used for anything but AQUA/LIQUID/VAPOUR? Since three fields already are populated from another enum (for compatibility reasons I presume), why not let the rest of the enum values be auto generated?

C++11 wise this should probably also be an enum class.